apis/management.cattle.io/v3public/authn_types.go

```diff
@@ -49,3 +49,11 @@ type ActiveDirectoryProvider struct {
 
 	DefaultLoginDomain string `json:"defaultLoginDomain,omitempty"`
 }
+
+type AzureADProvider struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+	AuthProvider      `json:",inline"`
+
+	Domain string `json:"domain,omitempty"`
```

Comment:
I cant find any instances of this field being read or written. Why do we have it?